Cargo.toml

```
@@ -34,6 +34,7 @@ base64 = "0.13"
 cargo-registry-s3 = { path = "src/s3", version = "0.2.0" }
 chrono = { version = "0.4.0", features = ["serde"] }
 civet = "0.12.0-alpha.4"
+clap = "3.0.0-beta.2"
```

```suggestion
clap = "=3.0.0-beta.2"
```

I'd pin the version since `clap` often introduces breaking changes in beta releases (https://github.com/clap-rs/clap/blob/master/CHANGELOG.md) and we could avoid updating it by mistake.

---

Sounds reasonable. I've changed it to pinned. 👍